src/main/java/net/atos/entng/support/services/impl/TicketServiceImpl.java

```diff
@@ -94,4 +94,12 @@ public Future<JsonObject> getSchoolWorkflowRightFromUserId(String userId, String
                 .onFailure(promise::fail);
         return promise.future();
     }
+
+    public Future<JsonObject> listStructureChildren(String school_id) {
+        Promise<JsonObject> promise = Promise.promise();
+        ticketServiceNeo4j.listStructureChildren(school_id)
```

Comment:
camelCase

Reply:
Je sais que la classe utilise un service TicketServiceNeo4jImpl mais toutes les méthodes sont en static...

Utilise la méthode en mode static 

```suggestion
      TicketServiceNeo4jImpl.getSchoolWorkflowRights(userId, workflowWanted, structureId)
```